src/utils.js

```diff
@@ -154,17 +160,21 @@ async function isDomainBlacklisted(domain) {
  * @param {string} url                          URL to call
  * @param {number|null} haveRedirectedTimes     Counter how many times we've redirected already
  * @param {object|null} headers                 Extra headers to use
+ * @param {boolean} checkBlacklist              Check if hostname of URL is on blaclist
  * @returns {Promise<object>}                   Response object
  * @throws                                      On non-20x response (after redirects) or a blacklisted domain
  */
-async function axiosGet(url, haveRedirectedTimes = null, headers = null) {
+async function axiosGet(url, haveRedirectedTimes = null, headers = null, checkBlacklist = 'true') {
```

Comment:
Could we make `checkBlacklist` a boolean and invert it into an explicit disable instead of an explicit enable?

```suggestion
async function axiosGet(url, haveRedirectedTimes = null, headers = null, disableBlacklistCheck = false) {
```